src/Servers/Kestrel/Core/src/Internal/Http3/Http3FrameWriter.cs

```diff
@@ -55,9 +56,16 @@ public Http3FrameWriter(PipeWriter output, ConnectionContext connectionContext,
             _memoryPool = memoryPool;
             _log = log;
             _streamIdFeature = streamIdFeature;
+            _http3Stream = http3Stream;
             _outgoingFrame = new Http3RawFrame();
             _flusher = new TimingPipeFlusher(_outputWriter, timeoutControl, log);
             _headerEncodingBuffer = new byte[_maxFrameSize];
+            _qpackEncoder = new QPackEncoder();
+
+            // TODO(JamesNK): QPack encoder doesn't react to settings change during a stream
```

Comment:
I'm not concerned about this.
- Settings rarely change after the start of a connection.
- Response header size limits are a best-effort requirement in the spec, the client is expected to "discard responses that it cannot process."

Remove the TODO but keep the comment.